ydb/docs/en/core/reference/ydb-sdk/grpc-headers.md

```diff
@@ -0,0 +1,17 @@
+## gRPC metadata headers
+
+{{ ydb-short-name }} uses next gRPC metadata headers:
+
+	- gRPC headers which client send to {{ ydb-short-name }}:
+	  - `x-ydb-database` - database
+	  - `x-ydb-auth-ticket` - auth ticket from credentials provider
+    - `x-ydb-sdk-build-info` - {{ ydb-short-name }} SDK build info
+    - `x-ydb-request-type` - optional user-defined header for mark client request 
+	  - `x-ydb-trace-id` - deprecated since {{ ydb-short-name }} 24.1
+    - `x-ydb-application-name` - optional user-defined application name
+	  - `x-ydb-client-capabilities` - supported client SDK capabilities (`session-balancer` and other)
+	  - `x-ydb-client-pid` - client application process ID
+
+	- gRPC headers which {{ ydb-short-name }} send to client with trsponses:
+	  - x-ydb-server-hints` - notifications from {{ ydb-short-name }} (such as `session-close` and other)
+	  - `x-ydb-consumed-units` - consumed units on current request
```

Comment:
```suggestion
    - `x-ydb-request-type` - optional user-defined header for marking client request 
	  - `x-ydb-trace-id` - deprecated since {{ ydb-short-name }} 24.1
    - `x-ydb-application-name` - optional user-defined application name
	  - `x-ydb-client-capabilities` - supported client SDK capabilities (`session-balancer` and other)
	  - `x-ydb-client-pid` - client application process ID

	- gRPC headers which {{ ydb-short-name }} sends to client with responses:
	  - x-ydb-server-hints` - notifications from {{ ydb-short-name }} (such as `session-close` and other)
	  - `x-ydb-consumed-units` - consumed units on the current request
```